### PR TITLE
Remove coveralls

### PR DESCRIPTION
**Pull request by Yozhef**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,7 +37,6 @@
         }
     },
     "scripts": {
-        "test": "phpunit",
```
Comment (shadowhand) [inline composer.json:41]: Why is this removed?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -21,8 +21,6 @@
         </whitelist>
     </filter>
     <logging>
-        <log type="coverage-text" target="php://stdout"/>
-        <log type="coverage-clover" target="build/logs/clover.xml"/>
-        <log type="coverage-html" target="build/coverage" charset="UTF-8" yui="true" highlight="true"/>
+        <log type="coverage-clover" target="clover.xml"/>
```
Comment (shadowhand) [inline phpunit.xml.dist:24]: This path needs to continue to be under `build/` **or** `clover.xml` needs to be added to `.gitignore`.

**Review by shadowhand — CHANGES_REQUESTED** [review]
Again, mixing types with other changes.

File: composer.json (hunk 1 above)
Comment (Yozhef) [reply composer.json:41]: because now need run 2 type PHPUnit 
1) only run PHPUnit
2) run PHPUnit and track code coverage

Comment (shadowhand) [reply composer.json:41]: Why not configure php to always run with coverage?

**Comment by shadowhand** [comment]
@Yozhef needs rebase.

File: composer.json (hunk 1 above)
Comment (Yozhef) [reply composer.json:41]: @shadowhand we can do it but I don't see the point in every version of the PHP track

Comment (shadowhand) [reply composer.json:41]: Oh, right. Forgot we are testing many versions.

**Review by shadowhand — APPROVED** [review]
